domains/t3vada.json

```diff
@@ -3,6 +3,7 @@
     "username": "wythh24"
   },
   "record": {
-    "CNAME": "devada.vercel.app"
+    "CNAME": "devada.vercel.app",
+    "TXT": ["google-site-verification=PGU03un4k5892v3xMTy8Tnz10tmSGTMJQ4V2ZNv65qQ"]
```

Comment:
you cannot use any other records with `CNAME`

Reply:
what @21Z said